third_party/nvfuser/csrc/fusion.h

```diff
@@ -53,6 +53,7 @@ class WelfordResult;
 
 class SegmentCandidateFinder;
 class SegmentedFusion;
+class MultiGroupFusion;
```

Comment:
I don't think this should be changes in this file.

Reply:
I don't like the idea of changes to Fusion with this PR. I don't think it should be necessary, but may need to review more.